app-mobilephone/gammu/gammu-1.38.4.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Please also add yourself as a proxied maintainer and proxy-maint project to back you up.